labgrid/driver/onewiredriver.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import requests
+
+import attr
+
+from ..factory import target_factory
+from ..resource import OneWirePort
+from .common import Driver
+
+@target_factory.reg_driver
+@attr.s
+class OneWireDriver(Driver):
```

Comment:
Corresponding to the change to the Protocol, this should be a OneWirePIODriver, implementing the DigitalOutput protocol.